lib/Mojo/Util.pm

```diff
@@ -23,6 +23,11 @@ use Unicode::Normalize ();
 # Check for monotonic clock support
 use constant MONOTONIC => eval { !!Time::HiRes::clock_gettime(Time::HiRes::CLOCK_MONOTONIC()) };
 
+# For speeding up @{ $foo // [] }
+use constant _empty => [];
+# make sure nobody accidentally modifies the referenced array
```

Comment:
Inconsistent comment.

Reply:
Reworded (also expanded the first comment a bit)